packages/popover/tsconfig.json

```diff
@@ -19,17 +19,14 @@
     {
       "path": "../lib"
     },
-    {
-      "path": "../leafygreen-provider"
-    },
-    {
-      "path": "../palette"
```

Comment:
ditto

Reply:
Updates the depcheck script to add packages back as `devDependencies` if they're used in `.story` or `.spec`